### PR TITLE
Fixes #27810 - Fixed inconsistent in output format

### PR DESCRIPTION
**Pull request by yifatmakias**

### CODE REVIEW

**Comment by yifatmakias** [comment]
[ test ]

File: lib/hammer_cli_foreman_virt_who_configure/config.rb [hunk 1]
```diff
@@ -117,7 +117,7 @@ class FetchCommand < HammerCLIForeman::Command
 
       failure_message _('Could not fetch the Virt Who configuration')
 
-      option ['--output', '-o'], 'FILE', _('File where the script will be written.')
+      option ['-o', '--output'], 'FILE', _('File where the script will be written.')
```
Comment (ofedoren) [inline lib/hammer_cli_foreman_virt_who_configure/config.rb]: Option description must not end with a dot. It's here for a while and I think it's time to fix it as well.

**Review by ofedoren — CHANGES_REQUESTED** [review]
Thanks, @yifatmakias!

LGTM, but just a small nitpick (see inline).

File: lib/hammer_cli_foreman_virt_who_configure/config.rb (hunk 1 above)
Comment (yifatmakias) [reply lib/hammer_cli_foreman_virt_who_configure/config.rb]: Thanks, I removed the dot.

**Review by ofedoren — APPROVED** [review]
Thanks, @yifatmakias! Merging.

**Pull request merged**